.travis.yml

```diff
@@ -182,7 +182,7 @@ before_install:
   - travis_retry pip install --upgrade pip setuptools wheel
   - travis_retry pip install cython
   - if [ -n "$NUMPYSPEC" ]; then travis_retry pip install $NUMPYSPEC; fi
-  - travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
+  - travis_retry pip install --upgrade pytest==5.0.1 pytest-xdist mpmath argparse Pillow codecov
```

Comment:
If it does work, it would be nice to pin it just for the 3.5 build that was failing

Reply:
Right, perhaps conditional on `USE_DEBUG`; it may be debug mode Python rather than 3.5 itself that is the culprit..

Reply:
The problematic job is already passing CI so I've gone ahead and tried to isolate the pytest version pin to 3.5-debug Python only. I think pip is smart enough to get rid of the new version with the subsequent call to install an older version.